### DISCUSSION TITLE
TFC Asia-Pacific

### DISCUSSION THREAD

**Discussion by luke026** [iptv-org/community#660] [Q&A]
**Channel Name:** tfc 
**Channel ID:** TheFilipinoChannelAsia.us
**Country:** phillipine
**Language:** tagalog
**External Link (optional):**tfc
**Notes (optional):** can u give this link in iptv ios

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19661
